test/lint/lint-circular-dependencies.py

```diff
@@ -37,7 +37,7 @@ def main():
 
     os.chdir(CODE_DIR)
     files = subprocess.check_output(
-        ['git', 'ls-files', '--', '*.h', '*.cpp'],
+        ["git", "ls-files", "--", "*.h", "*.cpp"],
```

Comment:
In 9ff4bb56baa509c587698cb3a2877dfd4ff6d136:
This seems unrelated to this commit, and is not mentioned in the commit message.